.github/workflows/swift.yaml

```diff
@@ -0,0 +1,106 @@
+name: Swift CI
+
+on:
+  push:
+    branches:
+      - master
+      - '!gh-pages'
+    paths:
+      - '*.podspec'
+      - 'Gemfile*'
+      - 'Package.swift'
+      - 'swift/**'
+      - '.github/workflows/swift.yaml'
+  pull_request:
+    paths:
+      - '*.podspec'
+      - 'Gemfile*'
+      - 'Package.swift'
+      - 'swift/**'
+      - '.github/workflows/swift.yaml'
+
+jobs:
+  development-apps:
+    runs-on: macOS-latest
+
+    strategy:
+      matrix:
+        scheme: 
+        - Development-Unit-WorkflowTests
+        - Development-Unit-WorkflowUITests
+        - Development-Unit-SplitScreenTests
+        - Development-Unit-TicTacToeTests
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install Bundler
+      run: |
+        gem update --system
+        gem install bundler
+
+    - name: Bundle Install
+      run: bundle install
+
+    - name: Pod Install
+      run: |
+        bundle exec pod gen Development.podspec
+
+    - name: Switch Xcode
+      run: sudo xcode-select -s /Applications/Xcode_11.3.1.app
+
+    - name: Build & Test
+      run: |
+        set -o pipefail && xcodebuild -workspace gen/Development/Development.xcworkspace -scheme ${{ matrix.scheme }} -destination platform\=iOS\ Simulator,OS\=13.3,name\=iPad\ Pro\ \(9.7-inch\) build test | xcpretty
+
+  spm:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Swift Package Manager - iOS
+      run: |
+        xcodebuild -scheme "Workflow-Package" test -destination "name=iPhone 11"
```

Comment:
Shouldn't this be using SPM to build/test?

Reply:
Hmm, actually I don’t think we can since WorkflowUI depends on UIKit, which SPM doesn’t know about (in my experiment locally `swift test` works just fine but it skips all of the WorkflowUI tests because `#if canImport(UIKit)`.

Reply:
(Now that WorkflowUI is pretty useless without UIKit, we should probably make that a hard dependency…)